apps/api/base/apisix.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: repo-apisix
+spec:
+  interval: 5m
+  url: https://charts.apiseven.com
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: apisix
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: apisix
+      version: 2.9.0
+      sourceRef:
+        kind: HelmRepository
+        name: don-api-repo-apisix
+  values:
+    host: set-in-overlay
+    replicaCount: 1
+
+    gateway:
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 9080
+      https:
+        enabled: true
+        servicePort: 9443
+    ingress:
+      ingress:
+        enabled: true
+        hostname: set-in-overlay
+        tls: true
+        annotations:
+          cert-manager.io/cluster-issuer: letsencrypt-prod
+
+    admin:
+      admin_key:
+        - name: "admin"
+          key:
+            valueFrom:
+              secretKeyRef:
+                name: apisix
+                key: apisix-admin
+          role: admin
+
+    etcd:
```

Comment:
Setting is anders in de Helm chart. Is denk ik in ons geval niet nodig, aangezien `etcd` door de Helm chart al intern wordt gemanaged.